src/main/java/org/opensearch/ad/transport/handler/AnomalyResultBulkIndexHandler.java

```diff
@@ -42,15 +42,24 @@ public class AnomalyResultBulkIndexHandler extends AnomalyIndexHandler<AnomalyRe
     private AnomalyDetectionIndices anomalyDetectionIndices;
 
     public AnomalyResultBulkIndexHandler(
-        SDKRestClient client,
+        SDKRestClient sdkRestClient,
         Settings settings,
         ThreadPool threadPool,
         ClientUtil clientUtil,
         IndexUtils indexUtils,
         SDKClusterService clusterService,
         AnomalyDetectionIndices anomalyDetectionIndices
     ) {
-        super(client, settings, threadPool, ANOMALY_RESULT_INDEX_ALIAS, anomalyDetectionIndices, clientUtil, indexUtils, clusterService);
+        super(
+            sdkRestClient,
+            settings,
+            threadPool,
+            ANOMALY_RESULT_INDEX_ALIAS,
+            anomalyDetectionIndices,
+            clientUtil,
+            indexUtils,
+            clusterService
```

Comment:
missed a change to `sdkClusterService` here

Reply:
ACK